gulpfile.babel.js

```diff
@@ -39,11 +39,11 @@ const $ = gulpLoadPlugins();
 const reload = browserSync.reload;
 
 // Lint JavaScript
-gulp.task('jshint', () =>
+gulp.task('eslint', () =>
```

Comment:
Yes, you are right. But I think, maybe, it would be better `scripts:lint`, because CSS/SASS linters are beginning to make popular.